tests/test_pumptrap_calibration.py

```diff
@@ -107,7 +107,8 @@ def test_tpump_analysis():
                         cs_fit_thresh = cs_fit_thresh, 
                         input_T=input_T,
                         bins_E=bins_E, bins_cs=bins_cs)
-
+    # filename check
+    assert tpump_calibration.filename.endswith('TPU_CAL.fits')
```

Comment:
Can we check that the filename also is based on the last file in the input dataset?